docs/release/osg-36.md

```diff
@@ -0,0 +1,48 @@
+OSG 3.6 News
+============
+
+**Supported OS Versions:** EL7, EL8
+
+The OSG 3.6 release series introduces HTCondor 8.9 and soon HTCondor 9.0.
+With HTCondor 9.0 we will be transistioning to token based authentication.
+We also dropped support for the GridFTP, GSI authentication, and Hadoop.
+
+To update to the OSG 3.6 series, please consult the page on
+[updating between release series](updating-to-osg-36.md).
+
+Latest News
+-----------
+
+### 2020-02-26: 3.6 Released
+
+This is the start of a new release series where we introduce major changes.
+One of the major changes is the shift to token based authentication.
+Here is a short list of the differences:
+
+-   GridFTP, GSI, and Hadoop are no longer available
+-   Added packages to support token-based authentication
+-   HTCondor 8.9.11 (HTCondor 8.9.12 release was delayed)
+-   XRootD 5.0.0 (XRootD 5.1.0 release was delayed)
+-   HTCondor 5.0.0 (HTCondor-CE 6.0.0 release was delayed)
+-   Gratia probe
+    -   consolidated all batch system probes into the HTCondor-CE probe
+    -   dropped all storage probes
+
```

Comment:
We should add a line item for osg-configure here

Reply:
@matyasselmeci any high-level things that you want to highlight here? It doesn't have to be as thorough as the upgrade notes

Reply:
- deprecated RSV config
- dropped unused configuration modules and attributes
- reorganized some configuration (see migration instructions)

Reply:
Added suggestion https://github.com/opensciencegrid/docs/pull/790/files#r583835583